src/main/webapp/timeseries/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+    font: 16px 'Open Sans';
+    color: #2C3E50;
+}
+a {
+    text-decoration: none;
+    color: #3498DB;
+}
+.container {
+    margin: 1%;
+}
+.text {
+    width: 100%;
+}
+.text p {
+    width: 75%;
+    margin: auto;
+    padding: 10px 0px;
+    text-align: center;
+}
+code {
+    width: 95%;
+    margin: auto;
+    text-align: center;
+}
+@media screen and (min-width: 800px) {
+    code {
+        width: 70%;
+    }
+}
+.header {
+    width: 100%;
+    text-align: center;
+    margin: 1% 0;
+}
+.header img {
+    padding-top: 10px;
+}
+.header div {
+    font-weight: 700;
+    font-size: 1.6em;
+    padding-top: 10px;
+}
+.dac-logo {
+    display: none;
+}
+@media screen and (min-width: 800px) {
+    .dac-logo {
+        display: block;
+        position: absolute;
+        left: 5px;
+        top: 5px;
+    }
+    .dac-logo img {
+        width: 120px;
+    }
+}
+.contact {
+    width: 50%;
+    margin: auto;
+    text-align: center;
+}
+.footer {
+    font-size: 0.9em;
+    text-align: center;
+    padding: 5px;
+}
+/** Styles for TIMESERIES */
+
+.timeseries {
+    width: 100%;
+    padding: 2% 0;
+}
+.circ {
+    fill: #D35400;
+    opacity: 0.6;
+    stroke: rgba(211, 84, 0, 0.7);
+    stroke-width: 5px;
+}
+.domain {
+    display: none;
+}
+.axis path, .axis line {
+    fill: none;
+    stroke: #000;
+    shape-rendering: crispEdges;
+}
+.axis text {
+    font-size: 12px;
+}
+.tick line {
+    stroke: #d5d5d5;
+    opacity: 0.7;
+}
+.brush .extent {
+    stroke: #fff;
+    fill-opacity: .125;
+    shape-rendering: crispEdges;
+}
+.clear-brush {
+    display: none;
+}
+.brush-control {
+    padding: .5%;
+    width: 100%;
+    text-align: center;
+}
+.brush-info {
+    display: inline-block;
+    font-size: 14px;
+}
```

Comment:
Add a newline here.

Reply:
Done, but if we are doing the bargraph we don't need this